include/depthimage_to_laserscan/DepthImageToLaserScan.h

```diff
@@ -27,24 +27,27 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 
-/* 
+/*
```

Comment:
I don't think we should necessarily lint this, so that we can pull updates from upstream later (if necessary).  The only formatting changes I did were whitespace, which I'll probably remove.